.github/workflows/build.yml

```
@@ -118,7 +118,7 @@ jobs:
     strategy:
       matrix:
         java: [ 11, 17 ]
-        junit-version: [ '5.8.2' ]
+        junit-version: [ '5.8.2', '5.9.0' ]
```

You can / have to remove 5.8.2, as our test code is now incompatible.

```suggestion
        junit-version: [ '5.9.0' ]
```